DFC.App.Account/Views/ConfirmDelete/Body.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@model DFC.App.Account.ViewModels.ConfirmDeleteCompositeViewModel
+
+<div class="govuk-main-wrapper govuk-main-wrapper--auto-spacing">
```

Comment:
Please remove this line as it will cause an issue with the spacing of breadcrumbs.

Reply:
Fixed